source/js/view.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * Created by Ksenia Annenko on 24.02.2017.
+ */
+
+/********************** View **********************/
+"use strict";
+
+let viewModel = new ViewModel();
+let occupiedCells;
+let playField = document.getElementById('play_field');
+let rowsInput = document.getElementById('rowsInput');
+let columnsInput = document.getElementById('columnsInput');
+let winLengthInput = document.getElementById('winLengthInput');
+let playerInstanceBlock = document.getElementById('player_instance_block');
+let cells = document.getElementsByClassName("cell");
+let availableSymbolList = viewModel.getAvailableSymbolList();
+
+/************ Starting game ************/
+window.onload = function() {
+    let list = viewModel.getPlayerList();
+    rowsInput.value = viewModel.getRows();
+    columnsInput.value = viewModel.getColumns();
+    winLengthInput.value = viewModel.getWinLength();
+
+    for (let i = 0; i < list.length; i++) {
+        getPlayerInstanceTemplate(list[i]);
+    }
+};
+
+function getPlayerInstanceTemplate(player) {
```

Comment:
Can we give this function a better name? It does not return anything so it shouldn't start from "get". Also it works with template but the template is not the end result of its execution. Maybe addPlayerInstance would fit better?